### DISCUSSION TITLE
[Feature] Partner Sharing from a specific date

### DISCUSSION THREAD

**Discussion by rgareh** [immich-app/immich#13668] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I want to be able to keep all my old photos that my partner would not be interested in, but currently partner sharing is all or nothing.

If we could select share photos from a certain date with partner, it would fix that.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#2654

As requested in the template, please search for duplicates.
